doc/manual/manual.tex

```diff
@@ -1595,14 +1741,198 @@ \subsubsection{Compressible models}
 where $\frac{1}{\rho} \frac{\partial \rho}{\partial p}$ is often referred to
 as the compressibility.
 
-In the implementation used in \aspect{}, this equation replaces
+For this approximation, this equation replaces
 \eqref{eq:stokes-2}. It has the advantage that it retains the symmetry of the
```

Comment:
There's no *equation* above, just an *identity*. The equation would require that the term be zero.

But it is also not symmetric to the other term, only if you bring the second term to the rhs.